akka-docs/src/test/java/jdocs/stream/operators/SinkDocExamples.java

```diff
@@ -6,23 +6,26 @@
 
 import akka.NotUsed;
 import akka.actor.ActorSystem;
+import akka.japi.Pair;
```

Comment:
It might be nice to include the 'relevant' imports into the example, by adding `//#takeLast-operator-example` around them as well. Not essential for this PR though.

Reply:
done.